package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@daostack/subgraph",
-  "version": "0.0.28",
+  "version": "0.0.29",
```

Comment:
why increment version ?

Reply:
Because we said we want to release a new package.